tigrex/__main__.py

```diff
@@ -61,40 +64,61 @@ def search(*args):
         Card.__print_card_header(response, card_layout)
         Card.__print_card_search(response, card_layout)
 
+    @staticmethod
+    def currency(*args):
+        if not len(args) == 1:
+            return "Please only provide one value for the currency"
+        if args[0] == 'eur':
```

Comment:
@Shadesfear, I like to create constants for Strings because it can lead to magic strings lingering around. I know it may seem trivial, but it's a thing I like to practice. So items such as 'eur', 'usd', 'tix', 'Price', 'sign' should all be encapsulated inside constant variables just incase something changes later down the line.

Reply:
Fixed